pkgs/tools/security/semgrep/default.nix

```diff
@@ -21,9 +21,10 @@ buildPythonApplication rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "typing-extensions~=4.2" "typing-extensions" \
+      --replace "attrs~=21.3" "attrs" \
+      --replace "boltons~=21.0" "boltons" \
       --replace "jsonschema~=3.2" "jsonschema" \
-      --replace "boltons~=21.0" "boltons"
+      --replace "typing-extensions~=4.2" "typing-extensions" \
```

Comment:
Would you mind migrating this to `pythonRelaxDepsHook`?

Reply:
sure I'll take a look at how to do that

Reply:
done